benchmarks/pthreadsbench.c

```diff
@@ -48,13 +51,14 @@ int main(int argc, char *argv[]) {
 
     SHARED_OBJECT_INIT();
```

Comment:
In fadbench.c this function is removed.
Consider making the code uniform. Maybe a todo for another PR?

Reply:
This function was used in previous versions of the code. Thus, I'm trying to remove it from everywhere.

Reply:
Removed.